docs-src/luna-streaming-core/modules/ROOT/nav.adoc

```diff
@@ -6,7 +6,7 @@
 ** xref:quickstart-server-installs.adoc[Install on Server/VM]
 * Guides and Examples
 ** xref:admin-console-tutorial.adoc[Pulsar Admin Console]
-** xref:admin-console-VM.adoc[Install Admin Console on Server/VM]
+** xref:admin-console-vm.adoc[Install Admin Console on Server/VM]
```

Comment:
This, and all other xrefs to `admin-console-vm.adoc`, are incorrect. That's because `VM` is capitalized in the name of the actual file.

Either change the name of the file so that `vm` is lowercase, or change all your xrefs back to:

```suggestion
** xref:admin-console-VM.adoc[Install Admin Console on Server/VM]
```